public/app/features/query-library/api/user.ts

```diff
@@ -0,0 +1,18 @@
+import { getBackendSrv, config } from '@grafana/runtime';
+
+import { UserDataQueryResponse } from './types';
+
+/**
+ * @alpha
+ */
+export const API_VERSION = 'iam.grafana.app/v0alpha1';
+
+/**
+ * @alpha
+ */
+const BASE_URL = `apis/${API_VERSION}/namespaces/${config.namespace}/display`;
+
+export async function getUserInfo(url?: string): Promise<UserDataQueryResponse> {
+  const userInfo = await getBackendSrv().get(`${BASE_URL}${url}`);
```

Comment:
Is there a POST version of this? I mean usually, it's recommended to use POST where the query is big (like list of IDs) so I wonder if there is only GET version of this or not.

Reply:
There's no POST version of this. This service currently doesn't support any POST requests